sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -55,32 +60,53 @@ export class AssessmentAESTController extends BaseController {
    */
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
-    @Param("applicationId") applicationId: number,
+    @Param("applicationId", ParseIntPipe) applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    let requestAssessmentSummary: RequestAssessmentSummaryAPIOutDTO;
+    const offeringChange =
+      await this.educationProgramOfferingService.getOfferingRequestsByApplicationId(
+        applicationId,
+      );
+    if (offeringChange) {
+      requestAssessmentSummary = {
+        id: offeringChange.id,
+        submittedDate: offeringChange.submittedDate,
+        status: OfferingStatus.ChangeAwaitingApproval,
+        requestType: RequestAssessmentTypeAPIOutDTO.OfferingRequest,
+        programId: offeringChange.educationProgram.id,
```

Comment:
@andrewsignori-aot @ann-aot @henryoforeh-aot 
For all the offering pages, we are doing a separate API call to get few properties of the program to which the offering belongs to perform offering related validation. This makes programId a dependency to all offering related pages.
We can create a tech debt to change the offering API to include the program related properties which are required to validate the offering form and get rid off this dependency.

Reply:
ya, we can add a TODO

Reply:
Created technical debt for the same. 
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1391